docs/cli_tools.rst

```diff
@@ -255,15 +311,15 @@ Example
 pt-brightness
```

Comment:
I think that deprecated commands shouldn't even be discussed in the docs. We should put any information that people using those old commands would need into the print-out when the user attempts to use them (`.deprecated`).

Reply:
We could print something like this:
```suggestion
To learn about the command arguments, check `pi-top display --help`.
```